src/Models/Currency.php

```diff
@@ -34,6 +34,10 @@ public static function globalCurrencies()
 
     public static function localForCountry($country)
```

Comment:
Can we change this method name to `Currency::getLocalByCountry` or something like this?